API/src/main/java/de/brandwatch/minianalytics/api/solr/model/Mention.java

```diff
@@ -16,16 +17,19 @@
     private String id;
 
     @Indexed(name = "queryID", type = "long")
+    @Expose
```

Comment:
What does `@Expose` do?

Reply:
It's an annotation for the Gson. When Gson converts a Object to a JSON, it will only convert all attributes annotated with `@Expose`.

Reply:
Maybe not the best idea to add gson into the mix with Spring using Jackson. But not a big deal either.